proposals/xxxx_carvel-package/README.md

```diff
@@ -0,0 +1,231 @@
+   that may be used to deliver software onto a Kuberentes cluster. See [the docs](https://carvel.dev/kapp-controller/docs/v0.47.x/packaging/#package)
+   for more information. 
+- `PackageMetadata` is a custom resource describing attributes for a `Package`.  See [the docs](https://carvel.dev/kapp-controller/docs/v0.47.x/packaging/#package-metadata)
+   for more information.
+
+## Proposal
+
+Allow Pinniped to be deployed onto a Kuberentes cluster through the mechanism of two Carvel `Packages`,
+a Supervisor and a Concierge package. These may be delivered via a `PackageRepository` resource and installed
+via `PackageInstall` custom resources, and `Secret`s containing `Package` configuration.
+
+Conceptually, cluster managers would make the Pinniped software available on the 
+cluster by deploying the PackageRepository:
+
+```bash
+# Deploy the Pinniped PackageRepository to the globally available
+# namespace watched by kapp-controller for new Packages 
+kapp deploy --app pinniped-package-repository --file <pinniped-release-files>/pinniped-package-repository.yaml
+```
+
+Then developers responsible for deploying Supervisor and Concierge would create the 
+appropriate resources to successfully deploy the PackageInstall and Packages for both 
+`Supervisor` and `Concierge`:
+
+```bash 
+# create a Service account and RBAC for the PackageInstall 
+vim supervisor-service-and-rbac.yaml
+kapp deploy --app supervisor-rbac --file supervisor-service-and-rbac.yaml
+vim concierge-service-and-rbac.yaml
+kapp deploy --app concierge-rbac --file concierge-service-and-rbac.yaml
+
+# create a PackageInstall and a Secret for configuring the Concierge
+vim supervisor-package-install-bundle.yaml
+kapp deploy --app supervisor --file supervisor-package-install-bundle.yaml
+vim concierge-package-install-bundle.yaml
+kapp deploy --app supervisor --file concierge-package-install-bundle.yaml
+```
+
+The `PackageRepository` will contain a series of versions of each of the Packages for Supervisor 
+and Concierge.
+
+The `PackageInstall` files will contain `constraints` representing acceptable versions of both the 
+`Supervisor` and `Concierge. For example:
+
+```yaml
+spec:  
+  packageRef:
+    # there will be two separate PackageInstall files, one for each 
+    # Supervisor and Concierge
+    refName: "supervisor.pinniped.dev"
+    versionSelection:
+      # Constraints may be used to specify an exact version of the package    
+      constraints: "0.25.0"  
+      # Alternatively, a constraint can be based on a semver range and can 
+      # specify multiple acceptible versions of the software.  In this case, 
+      # the Package will automatically upgrade to new versions when they become
+      # available, for example, when a new verison of the PackageRepository is 
+      # deployed containing new versions of the Packages.
+      constraints: ">0.25.0"  
+```
+
+### Goals and Non-goals
+
+Goals
```

Comment:
At this point, should we update this section to indicate that this is not a fully supported feature and is instead meant to provide users both an example and a mechanism they may use to create their own package?